### PR TITLE
GET /api/topics

### PR DESCRIPTION
**Pull request by Skahrr**
MVC set up + first endpoint done. 0 error handling because I can not think of a possible error made by the user if the url is correct.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -4,8 +4,32 @@ const request = require("supertest");
...
+            {
+              description: 'Not dogs',
+              slug: 'cats'
+            },
+            {
+              description: 'what books are made of',
+              slug: 'paper'
+            }
+          ]
+        expect(body.topics).toEqual(expected)
```
Comment (cavalierswain) [inline __tests__/app.test.js]: have a look at some testing methods ie. .toMatchObject or .objectContains, as these specially test objects. 

look at expect.any() to help make your testing flexible.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -4,8 +4,32 @@ const request = require("supertest");
...
+       })
```
Comment (cavalierswain) [inline __tests__/app.test.js]: you can also test the length here too!

**Comment by cavalierswain** [comment]
left a few comments in the test file, have a think about them and then merge away! 👍

**Comment by Skahrr** [comment]
I've pushed the changes to the test file, does it look better now?

**Pull request merged**